+dj/Schema.m

```diff
@@ -32,6 +32,7 @@
         %   computed: tableName with '__'
         allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job'}
         tierPrefixes = {'#', '', '_', '__', '~'}
+        tierClasses = {'dj.Lookup' 'dj.Manual' 'dj.Imported' 'dj.Computed' 'dj.Jobs'}
```

Comment:
I much prefer that we place the `tierPrefix` information directly inside the corresponding class, rather than using the class as mere naming magic.